apps/blink/main.c

```diff
@@ -2,11 +2,13 @@
 #include <gpio.h>
 
 int main(void) {
-    gpio_enable_output(LED_0);
-
+    gpio_set(0);
     while(1) {
-      gpio_toggle(LED_0);
-      delay_ms(500);
+      gpio_toggle(0);
+//      printf("Before delay.\n");
+      delay_ms(1000);
```

Comment:
The changes to this file are _OK_, but I don't think there's any reason for them. Please at least remove the commented out code.